### PR TITLE
feature: swap DB type via env

### PR DESCRIPTION
**Pull request by InoMurko**
Slack.

## Overview

RocksDB is implemented, but not tested "in the wild".

## Changes

- Pull the DB type from ENV.

## Testing
Hey, If it works, it works.

### CODE REVIEW

File: apps/omg_db/config/config.exs [hunk 1]
```diff
@@ -3,7 +3,7 @@ use Mix.Config
 # see [here](README.md) for documentation
 
 config :omg_db,
-  type: :leveldb,
+  type: {:system, "DB_TYPE", :leveldb, {String, :to_atom}},
```
Comment (pdobacz) [inline apps/omg_db/config/config.exs]: I just spotted that this config var is undocumented, see https://github.com/omisego/elixir-omg#omgdb-configuration---omg_db-app

Comment (InoMurko) [reply apps/omg_db/config/config.exs]: The question is whether we want to put it there? It's not something we will support indefinitely.

Comment (InoMurko) [reply apps/omg_db/config/config.exs]: Type is only there to ease the transition, no?

Comment (pdobacz) [reply apps/omg_db/config/config.exs]: ah right, nvm then, haven't thought of that

**Pull request merged**